File: php/simple.php
<?php
// Simple PHP example
function greet($name) {
    return "Hello, $name!";
}

function farewell($name) {
    return "Goodbye, $name!";
}

function capitalize($name) {
    return ucfirst($name);
}

$message = greet("World");
echo $message . "\n";
echo farewell("World") . "\n";
echo capitalize("world") . "\n";
?>

File: python/simple.py
# Simple Python example
def greet(name):
    return f"Hello, {name}!"

def farewell(name):
    return f"Goodbye, {name}!"

def capitalize(name):
    return name.capitalize()

if __name__ == "__main__":
    message = greet("World")
    print(message)
    print(farewell("World"))
    print(capitalize("world"))
